crates/lakekeeper/src/request_metadata.rs

```diff
@@ -208,6 +209,18 @@ impl RequestMetadata {
     pub fn base_uri_management(&self) -> String {
         format!("{}/management", self.base_url())
     }
+
+    #[must_use]
+    pub fn refresh_credentials_endpoint(
+        &self,
+        warehouse_id: WarehouseId,
+        table: &TableIdent,
+    ) -> String {
+        format!(
+            "v1/{}/namespaces/{}/tables/{}/credentials",
+            warehouse_id, table.namespace, table.name,
```

Comment:
We can't use `Display` of the namespace. Instead, we should use `namespace.to_url_string()`.
Also the `table.name` might require URL encoding.